protocol.html

```diff
@@ -648,6 +654,34 @@ <h4 property="schema:name">Description Resource</h4>
                       <p><span about="" id="client-link-describes" rel="spec:requirement" resource="#client-link-describes">Clients can discover resources that are described by description resources by making an HTTP <code>HEAD</code> or <code>GET</code> request on the target URL, and checking the HTTP <code>Link</code> header with a <code>rel</code> value of <code>describes</code> (inverse of the <code>describedby</code> relation) [<cite><a class="bibref" href="#bib-rfc6892">RFC6892</a></cite>].</span></p>
                     </div>
                   </section>
+
+                  <section id="auxiliary-resources-stat-resource" inlist="" rel="schema:hasPart" resource="#auxiliary-resources-stat-resource">
+                    <h4 property="schema:name">Container Metadata Resource</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>
+			An auxiliary resource of type <em>Container Metadata Resource</em> provides server-maintained metadata about contained resources.
```

Comment:
```suggestion
			An auxiliary resource of type <em>Container Metadata Resource</em> provides server-maintained metadata about contained resources.
```
Shouldn't the type be a URI (presumably `#auxiliary-resources-stat-resource` with a base I'm not quickly seeing), rather than a literal?